src/octopeer-github/content/ContentController.ts

```diff
@@ -94,13 +143,17 @@ class ContentController {
      * Hook the different rawdata trackers to DOM.
      * @param database   the database that should be used when logging.
      */
-    private hookSyntacticToDOM(database: DatabaseAdaptable) {
-        // tslint:disable:no-unused-variable
-        let windowResolutionTracker = new WindowResolutionTracker(database);
-        let keystrokeTracker = new KeystrokeTracker(database);
-        let mouseClickTracker = new MouseClickTracker(database);
-        let mouseScrollTracker = new MouseScrollTracker(database);
-        let mousePositionTracker = new MousePositionTracker(database);
-        // tslint:enable:no-unused-variable
+    private hookTrackersToDOM(database: DatabaseAdaptable) {
+        const list: EventTracker[] = [
```

Comment:
I'm not sure, because the only place that this list is used, is right here. Plus, every time new Trackers should be instantiated, which means creating a new list. (of course it's a list because then the for-loop is an easy refactoring)